packages/blade-mcp/knowledgebase/components/TimePicker.md

```diff
@@ -0,0 +1,338 @@
+# TimePicker
+
+## Description
+
+The TimePicker component is a comprehensive time selection input that supports both 12-hour and 24-hour formats with configurable minute step intervals. It features an accessible segmented input field where users can directly type or use dropdown/bottom sheet selection with scrollable spin wheels. The component provides responsive layouts with desktop dropdown and mobile bottom sheet, includes validation states, and supports both controlled and uncontrolled usage patterns.
+
+## Important Constraints
+
+- `timeFormat` only accepts `'12h'` or `'24h'` values
+- `minuteStep` only accepts `1`, `5`, `15`, or `30` as valid values
+- `size` only accepts `'medium'` or `'large'` values
+- `labelPosition` only accepts `'top'` or `'left'` values
+- When using controlled mode, both `value` and `onChange` props must be provided
+- `onApply` callback is only triggered when `showFooterActions` is `true` and user clicks Apply button
+- Component requires a valid Date object or null for `value` and `defaultValue` props
+- `validationState` only accepts `'error'`, `'success'`, or `'none'` values
+
+## TypeScript Types
+
+These types define the props that the TimePicker component and its related components accept. Use these for proper TypeScript integration and to understand available configuration options.
+
+```typescript
+/**
+ * Time format types supported by TimePicker
+ * Both 12h and 24h formats are supported using React Aria.
+ */
+type TimeFormat = '12h' | '24h';
+
+/**
+ * Minute step intervals supported by TimePicker
+ */
+type MinuteStep = 1 | 5 | 15 | 30;
```

Comment:
why specific values instead of number? We only want these 1, 5, 15, 30 intervals?